nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -908,6 +909,7 @@ private ServerConnector createUnconfiguredSslServerConnector(Server server, Http
         final HttpConfiguration httpsConfiguration = new HttpConfiguration(httpConfiguration);
         httpsConfiguration.setSecureScheme("https");
         httpsConfiguration.setSecurePort(port);
+        httpConfiguration.setSendServerVersion(props.shouldSendServerVersion());
```

Comment:
Why is this variable `httpConfiguration` vs. `httpsConfiguration`?

Reply:
Fixed